Code.of.Conduct.DRiP.md

```diff
@@ -0,0 +1,61 @@
+---
+tags: 🌱/٤/🟩
+type: 🌱
+subject: "DRiP"
+subsubject: "Code-of-Conduct"
+status: 🟩
+last-reviewed: 2023-08-26
+creation-date: 2023-08-26
+author: Christophe Pere
+---
```

Comment:
Would it be possible to remove this meta-data from the file that is stored on the repo?